compass/ocean/mesh/lts_regions.py

```diff
@@ -0,0 +1,362 @@
+                                         [0.914, -1.012 + 2 * math.pi],
+                                         [0.850, -1.308 + 2 * math.pi],
+                                         [0.743, -1.293 + 2 * math.pi],
+                                         [0.638, -1.781 + 2 * math.pi],
+                                         [0.481, -1.737 + 2 * math.pi]])
+
+    eastUSCoastRegionPts = np.array([[0.532, 4.862],
+                                     [0.520, 4.946],
+                                     [0.523, 5.018],
+                                     [0.548, 5.082],
+                                     [0.596, 5.131],
+                                     [0.639, 5.159],
+                                     [0.690, 5.175],
+                                     [0.731, 5.168],
+                                     [0.760, 5.147],
+                                     [0.777, 5.148],
+                                     [0.790, 5.181],
+                                     [0.813, 5.048],
+                                     [0.556, 4.775]])
+
+    delawareCoastRegionPts = np.array([[0.598, 4.929],
+                                       [0.591, 4.953],
+                                       [0.591, 4.973],
+                                       [0.596, 4.991],
+                                       [0.607, 5.001],
+                                       [0.620, 5.005],
+                                       [0.633, 5.005],
+                                       [0.642, 4.999],
+                                       [0.683, 5.028],
+                                       [0.689, 5.046],
+                                       [0.697, 5.057],
+                                       [0.709, 5.064],
+                                       [0.719, 5.063],
+                                       [0.726, 5.059],
+                                       [0.727, 5.058],
+                                       [0.727, 5.053],
+                                       [0.730, 5.050],
+                                       [0.745, 5.011],
+                                       [0.654, 4.883]])
+
+    if region == 'east_us_coast':
+        fine_region = Polygon(eastUSCoastRegionPts)
+    elif region == 'western_atlantic':
+        fine_region = Polygon(westernAtlanticRegionPts)
+    elif region == 'delaware_coast':
+        fine_region = Polygon(delawareCoastRegionPts)
+    else:
+        print('Desired region is undefined')
+        quit()
+
+    return fine_region
+
+
+def label_mesh(fine_region, mesh, graph_info, num_interface):   # noqa: C901
+
+    # read in mesh data
+    ds = xarray.open_dataset(mesh)
+    nCells = ds['nCells'].size
+    nEdges = ds['nEdges'].size
+    areaCell = ds['areaCell'].values
+    cellsOnEdge = ds['cellsOnEdge'].values
+    edgesOnCell = ds['edgesOnCell'].values
+    latCell = ds['latCell']
+    lonCell = ds['lonCell']
+
+    # start by setting all cells to coarse
+    LTSRgn = [2] * nCells
+
```

Comment:
I'd like to keep all variables lowercase with underscores, even the mesh ones.

Reply:
Done with latest commit.